api/src/main/java/marquez/api/TagResource.java

```diff
@@ -51,6 +54,17 @@ public Response list(
     return Response.ok(new Tags(tags)).build();
   }
 
+  @Timed
+  @ResponseMetered
+  @ExceptionMetered
+  @PUT
+  @Consumes(APPLICATION_JSON)
+  @Produces(APPLICATION_JSON)
+  public Response create(@Valid Tag tag) {
```

Comment:
I'd use the path `/tags/{name}` with the HTTP body as the tag description for consistency with the other endpoints. And, agree that this can be a `POST`, but making it a `PUT` makes a lot of sense given that the caller is providing the unique identifier for the tag.

Reply:
The HTTP body still as json in the form
```
{
  "description" : "some string"
}
```
or plain text?

Reply:
Json, as it's likely that we'll add properties in the future.